### ISSUE TITLE
🌐 Website Issue | Merge several “software” sections

### ISSUE THREAD

**Issue by GrimPixel** [privacytoolsIO/privacytools.io#1427]
## Description
These pages can be merged to one:

Calendar/Contacts Sync Tools
File Sync
Self-Hosted Cloud Server

This has been talked about at https://github.com/privacytoolsIO/privacytools.io/issues/892

**Comment by danarel** [comment]
Not really. I mean, they share similarities, but people are often looking for those things individually and I believe listing them as such makes it easier for those users. File sync and self-hosting doesn't always mean calendar and contacts, or not at least to those without the knowledge to make them work that way.
